examples/multipass/app/components/Cart.tsx

```diff
@@ -65,7 +67,7 @@ function CartLines({
   layout,
 }: {
   layout: CartMainProps['layout'];
-  lines: OptimisticCartLine[];
+  lines: CartLine[];
```

Comment:
Needed to change, because just using the `OptimisticCartLine` directly without a generic means the default API query is used.